### ISSUE TITLE
Postponed release

### ISSUE THREAD

**Issue by jimangel** [kubernetes/sig-release#2392]
## Scheduled to happen: 

_Note for v1.x.0 releases: Having this issue in open state will stop the
periodic run of [`krel fast-forward`](https://testgrid.k8s.io/sig-release-releng-blocking#git-repo-kubernetes-fast-forward)
to avoid conflicts with releases running in parallel. The fast-forward will
automatically stop once the v1.x.0 tag is available._

## Release Blocking Issues

None

## Release Steps

- [x] Create a thread on #release-management: https://kubernetes.slack.com/archives/CJH2GBF7Y/p1702377440656929
- [x] Screenshot unhealthy release branch testgrid boards
- [x] Check the health of the publishing-bot  
- Mock Run
  - [x] Stage
  - [x] Release
- NoMock Run
  - [ ] Stage
  - [ ] Image Promotion: 
  - [ ] Release
- [ ] Notify #release-management: 
- [ ] Send notification: 
- [ ] Update [`schedule.yaml` file](https://github.com/kubernetes/website/blob/main/data/releases/schedule.yaml) with the latest release 
- [ ] Collect metrics and add them to the `Release steps` table below

## Release Tools Version

```
GitVersion:    v0.16.4
GitCommit:     unknown
GitTreeState:  unknown
BuildDate:     unknown
GoVersion:     go1.21.4
Compiler:      gc
Platform:      linux/amd64
```

## Release Jobs History

|     STEP     |                                             COMMAND                                              |                                                     LINK                                                      |             START              | DURATION | SUCCEEDED? |
|--------------|--------------------------------------------------------------------------------------------------|---------------------------------------------------------------------------------------------------------------|--------------------------------|----------|------------|
| `mock stage` | `krel stage --type official --branch release-1.29 --build-version v1.29.0-rc.2.3+ca9e7d9cd21646` | https://console.cloud.google.com/cloud-build/builds/834c8907-bada-4a1a-8383-f248a9960094?project=648026197307 | 2023-12-12T10:42:11.113366553Z | 01:07:26 | No         |
| `mock stage` | `krel stage --type official --branch release-1.29 --build-version v1.29.0-rc.2.3+ca9e7d9cd21646` | https://console.cloud.google.com/cloud-build/builds/1fb22ab5-6011-45a7-8f97-66de71e22023?project=648026197307 | 2023-12-12T11:55:20.125272291Z | 01:09:10 | Yes        |
| `mock release` | `krel release --type official --branch release-1.29 --build-version v1.29.0-rc.2.3+ca9e7d9cd21646` | https://console.cloud.google.com/cloud-build/builds/acafb672-e7ff-4ee1-8a3b-ff91fa204385?project=648026197307 | 2023-12-12T13:13:24.188663046Z | 00:17:28 | Yes        |
| Stage | `krel stage [arguments]` | | | | |
| Release | `krel release [arguments]` | | | | |

## Open Questions

/milestone v1.29
/assign @jimangel 
/cc @kubernetes/release-managers
/priority important-soon
/kind documentation

**Comment by saschagrunert** [comment]
Note: Release fast forward has successfully stopped because of this issue: https://storage.googleapis.com/kubernetes-jenkins/logs/ci-fast-forward/1734425747390468096/build-log.txt

```
Step #3: level=info msg="Skipping fast forward: release cut issue is open: https://api.github.com/repos/kubernetes/sig-release/issues/2392"
```

[`master`](https://github.com/kubernetes/kubernetes/commits/master) and [`release-1.29`](https://github.com/kubernetes/kubernetes/commits/release-1.29) branches are in sync / up-to-date

**Comment by saschagrunert** [comment]
### Testgrid dashboards for 1.29

Boards checked for FAILING:
- [sig-release-1.29-blocking](https://testgrid.k8s.io/sig-release-1.29-blocking)
- [sig-release-1.29-informing](https://testgrid.k8s.io/sig-release-1.29-informing)


**No FAILING Jobs**




**comment generated by [krel](https://github.com/kubernetes/release/tree/master/docs/krel)**

**Comment by saschagrunert** [comment]
Doing a performance benchmark for the release cut, here are the results:

| Step | Duration | Notes |
|:-|-:| :- |
| Issue creation, testgrid shots, slack announcement | 5min | |
| Mock stage (first try) | 1h 7min| |
| Mock release | | |
| NoMock stage (building release) | | |
| NoMock stage (building packages) | |  |
| Image promotion (time to merge) | |  |
| Image promotion (waiting for job) | | |
| Image promotion (job duration) | |  |
| NoMock release | | |

**Comment by saschagrunert** [comment]
Mock stage and release are done, handing back to @jimangel
